src/Components/LifecycleTable/LifecycleTable.tsx

```diff
@@ -238,16 +265,29 @@ export const LifecycleTable: React.FunctionComponent<LifecycleTableProps> = ({
           <Td dataLabel={APP_LIFECYCLE_COLUMN_NAMES.end_date}>{formatDate(repo.end_date)}</Td>
           {viewFilter !== 'all' && (
             <Td dataLabel={APP_LIFECYCLE_COLUMN_NAMES.count ?? 'N/A'}>
-              <Button
-                variant="link"
-                onClick={(event) => {
-                  handleModalToggle(event);
-                  setModalDataName(String(repo.name));
-                  setModalData(repo.systems);
-                }}
-              >
-                {repo.count}
-              </Button>
+              <div style={{ display: 'flex', alignItems: 'center', gap: '0' }}>
```

Comment:
```suggestion
              <div style={{ display: 'flex', alignItems: 'center', gap: '8px' }}>
```
based on the mocks the gap should be bigger

Reply:
I can adjust the leftPadding to match it